tests/integration_tests/test_pytorch_lightning.py

```diff
@@ -18,26 +18,30 @@ def __init__(self) -> None:
         super(Model, self).__init__()
         self._model = nn.Sequential(nn.Linear(4, 8))
 
-    def forward(self, data: torch.Tensor) -> torch.Tensor:
+    def forward(self, data: torch.Tensor) -> torch.Tensor:  # type: ignore
 
         return self._model(data)
 
-    def training_step(self, batch: List[torch.Tensor], batch_nb: int) -> Dict[str, torch.Tensor]:
+    def training_step(  # type: ignore
+        self, batch: List[torch.Tensor], batch_nb: int
+    ) -> Dict[str, torch.Tensor]:
 
         data, target = batch
         output = self.forward(data)
         loss = F.nll_loss(output, target)
         return {"loss": loss}
 
-    def validation_step(self, batch: List[torch.Tensor], batch_nb: int) -> Dict[str, torch.Tensor]:
+    def validation_step(  # type: ignore
+        self, batch: List[torch.Tensor], batch_nb: int
+    ) -> Dict[str, torch.Tensor]:
 
         data, target = batch
         output = self.forward(data)
         pred = output.argmax(dim=1, keepdim=True)
         accuracy = pred.eq(target.view_as(pred)).double().mean()
         return {"validation_accuracy": accuracy}
 
-    def validation_epoch_end(
+    def validation_epoch_end(  # type: ignore
```

Comment:
[note]

https://github.com/PyTorchLightning/pytorch-lightning/blob/80019874e57267e44d0eee9e898f51e45703e502/pytorch_lightning/core/lightning.py#L824

While `LightningModule.validation_epoch_end` returns nothing, `validation_epoch_end` here returns `Dict`

Reply:
I think we can suppress the error by updating the code instead of ignoring the type error.
According to the pytorch_lightning's code example, we can report the validation accuracy using `self.log`. So, how about replacing `return {"accuracy": accuracy}` with `self.log("accuracy", accuracy)`?

https://github.com/PyTorchLightning/pytorch-lightning/blob/80019874e57267e44d0eee9e898f51e45703e502/pytorch_lightning/core/lightning.py#L860-L866